azure_license-server/main.tf

```diff
@@ -0,0 +1,95 @@
+variable "azure_region" {}
+variable "resource_name" {}
+variable "public_key_location" {}
+
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "rg" {
+  name     = var.resource_name
+  location = var.azure_region
+  ssh_key = var.public_key_location
```

Comment:
This line causes an error:
```
│ Error: Unsupported argument
│
│   on main.tf line 12, in resource "azurerm_resource_group" "rg":
│   12:   ssh_key = var.public_key_location
│
│ An argument named "ssh_key" is not expected here.
```

I think it can be removed.